Command/CacheWarmupCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Go! AOP framework
+ *
+ * @copyright Copyright 2015, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\Symfony\GoAopBundle\Command;
+
+use Go\Core\AspectKernel;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Go\Console\Command\CacheWarmupCommand as BaseCommand;
+
+/**
+ * Class CacheWarmupCommand
```

Comment:
Could you please drop this this line as it doesn't provide any valuable information about this class. Same comment can be applied to all commands in this review.